### ISSUE TITLE
Document the integration process for LUA and WDL into demo 

### ISSUE THREAD

**Issue by RamarkoB** [uchicago-cs/chiventure#1543]
Building off of Issue #1524. There is previous documentation of Lua and WDL, but the Custom-Scripts team want to know how to store Lua information and use it using WDL. This issue will be marked closed when we have a working demo that uses Lua and WDL in conjunction, or we have an example that we can then integrate into a demo.

**Comment by RamarkoB** [comment]
The Custom-Scripts team had already developed a new demo to show off Boolean types in Lua and another Demo that showed off conditionals in WDL. I was able to combine the two into a single Demo that has Conditionals, alongside Lua Integration. Because there is another pull request issue that is integrating Lua and WDL into the demo, this issue will now center on documenting what we have learned from integrating Lua and the WDL.
